integration/containerd/runk/runk-tests.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+#
+# Copyright (c) 2021-2022 Kata Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This test will validate runk with containerd
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+KATACONTAINERS_REPO=${katacontainers_repo:="github.com/kata-containers/kata-containers"}
+RUNK_SRC_PATH="${GOPATH}/src/${KATACONTAINERS_REPO}/src/tools/runk"
+RUNK_BIN_PATH="/usr/local/bin/runk"
+TEST_IMAGE="docker.io/library/busybox:latest"
+CONTAINER_ID="id1"
+
+install_runk() {
+    echo "Install runk"
+    pushd ${RUNK_SRC_PATH}
+    make
+    sudo make install
+    popd
+}
+
+setup() {
+    echo "restart containerd service"
+    sudo systemctl restart containerd
+    echo "pull container image"
+    sudo ctr images pull ${TEST_IMAGE}
+}
+
+test_runk() {
+    echo "start container with runk"
+    sudo ctr run --pid-file ${CONTAINER_ID}.pid --rm -d  --runc-binary ${RUNK_BIN_PATH} ${TEST_IMAGE} ${CONTAINER_ID}
+    read CID PID STATUS <<< $(sudo ctr t ls | grep ${CONTAINER_ID})
+    [ ${PID} == $(cat ${CONTAINER_ID}.pid) ] || die "pid is not consistent"
+    [ ${STATUS} == "RUNNING" ] || die "contianer status is not RUNNING"
+
+    echo "exec process in a container"
+    sudo ctr t exec -d --exec-id id1  ${CONTAINER_ID} sh -c "while true; do echo hello; sleep 1; done"
```

Comment:
It's better to run exec and get some results. For example:
```bash
ctr exec touch /tmp/foo
ctr exec ls /tmp/foo
```
create one file in the first exec and check it existence in the second exec.